tools/tools.yml

```diff
@@ -138,33 +153,40 @@
       site: https://vtk.org/
       conda_channel: conda-forge
       badges: travis, pypi, conda, site
+      builton: vtk
 
     - repo: vispy/vispy
       conda_channel: conda-forge
       site: http://vispy.org
       badges: travis, appveyor, coveralls, pypi, conda, site
+      builton: opengl
 
     - repo: maartenbreddels/ipyvolume
       conda_channel: conda-forge
+      builton: opengl, webgl
 
     - repo: enthought/mayavi
       sponsors: [enthought]
       site: https://docs.enthought.com/mayavi/mayavi
       appveyor_project: EnthoughtOSS/mayavi
       badges: travis, appveyor, codecov, pypi, conda, site
+      builton: vtk, opengl
 
     - repo: pyvista/pyvista
       site: https://docs.pyvista.org
       conda_channel: conda-forge
+      builton: vtk
 
     - repo: InsightSoftwareConsortium/itkwidgets
       pypi_name: itkwidgets
       conda_package: itkwidgets
       conda_channel: conda-forge
       badges: pypi, conda, circleci
+      builton: vtk
```

Comment:
This is actually `webgl` instead of `vtk`.

Reply:
Fixed now; thanks!